stan-dev/cmdstan
# Issue/660 makefile for cmdstan manual

#### Submisison Checklist


#### Summary:
Removed dependency on makefile `stan/make/manual`, see issue #660.

#### Intended Effect:
Allow removal of all docs and assosicated make tasks from `stan` repo, as the Stan documentation has been moved over to the `docs` repo  (issue: https://github.com/stan-dev/stan/issues/2706)

#### How to Verify:

- run task `make clean-all`, verify no top-level `doc` dir
- run task `make manual`, verify that top-level `doc` dir exists, contains versioned CmdStan guide (pdf).

#### Side Effects:
N/A

#### Documentation:
N/A

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
